docs/reference/announcements-release-notes/880/880-announcements.md

```diff
@@ -342,3 +342,9 @@ Assignees list removed from response.
 These endpoints are superseded by [usage metrics endpoint](../../../apis-tools/orchestration-cluster-api-rest/specifications/get-usage-metrics.api.mdx).
 
 Operate and Tasklist usage metrics endpoints are **scheduled for removal in the 8.10 release**.
+
+### Process instance tags <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span><span class="badge badge--medium" title="This feature affects APIs">API</span> {#process-instance-tags}
+
+Camunda 8.8 adds optional, immutable **tags** for process instances you can set at creation via the Orchestration Cluster API or Clients and are also forwarded to job workers. Tags provide lightweight, structured metadata for routing, correlation, prioritization and more without inspecting full variable payloads.
+
+For a full feature overview see [process instance creation](/components/concepts/process-instance-creation.md#tags) and [job workers](/components/concepts/job-workers.md#tags).
```

Comment:
💭 I would suggest removing tags from this page as it's about breaking changes and env changes